packages/p2p/src/components/buy-sell/buy-sell.jsx

```diff
@@ -54,7 +54,7 @@ class BuySell extends Component {
                         name='filter'
                         onChange={this.onChangeTableType}
                         value={table_type}
-                        rounded_button
+                        rounded_button={true}
```

Comment:
No need to pass `true`, `rounded_button` will suffice.

```suggestion
                        rounded_button
```